<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IstDialog</class>
 <widget class="QDialog" name="IstDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonIstVideoOpen">
   <property name="geometry">
    <rect>
     <x>959</x>
     <y>461</y>
     <width>279</width>
     <height>99</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>视频开</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonIstVideoClose">
   <property name="geometry">
    <rect>
     <x>959</x>
     <y>461</y>
     <width>279</width>
     <height>99</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>视频关</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonIstHangUp">
   <property name="geometry">
    <rect>
     <x>959</x>
     <y>677</y>
     <width>279</width>
     <height>99</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>挂断</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonIstOpen">
   <property name="geometry">
    <rect>
     <x>959</x>
     <y>569</y>
     <width>279</width>
     <height>99</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>开锁</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonIstAnswer">
   <property name="geometry">
    <rect>
     <x>959</x>
     <y>353</y>
     <width>279</width>
     <height>99</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>接听</string>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>30</y>
     <width>858</width>
     <height>745</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameVideoShow">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>119</y>
     <width>831</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="ScrollLineEdit" name="lineEditInfo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>831</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22px;</string>
   </property>
   <property name="text">
    <string>通话中：1110-2222 陈明适</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabel" name="labelExtension1">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>120</y>
     <width>241</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 23px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>扩展分机1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabel" name="labelExtension2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>170</y>
     <width>241</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 23px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>通话中</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ScrollLineEdit</class>
   <extends>QLineEdit</extends>
   <header>scrolllineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonIstAnswer</tabstop>
  <tabstop>pushButtonIstVideoClose</tabstop>
  <tabstop>pushButtonIstVideoOpen</tabstop>
  <tabstop>pushButtonIstOpen</tabstop>
  <tabstop>pushButtonIstHangUp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
